docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/01_instrument_intro_prerequisites.md

```diff
@@ -0,0 +1,57 @@
+
+**PHP** instruments are forms coded manually by a developer in the PHP language. These instruments can also have rules and scoring defined.
+
+Choosing the format in which and instrument is coded is necessary before beginning the process. PHP instruments can easily be converted to LINST but the opposite is not true. Once an instrument is implemented in LINST format, transforming it to PHP is not trivial. Here are a few bullet points that can help you make your decision.
+
+- ***LINST***
+ - *Pros:*
+     - Easy to implement (LORIS Instrument Builder)
+     - Ideal for small instruments
+     - Less prone to bugs in the long term 
+     - Can be combined with rules and scoring 
+ - *Cons:*
+     - Rules and scoring logic can be limited in complex cases
+     - Rules and scoring might still require a developer
+     - Does not offer flexibility (default LORIS form elements are used)
+- ***PHP***
+ - *Pros:*
+     - Very flexible as form elements can be overwritten, implemented or customized
+     - allows for more complex validation and scoring
+     - Can easily be converted back to LINST
+ - *Cons:*
+     - Requires a developer
+     - Significant maintenance required over time to adapt to new LORIS releases
+     - less stable as some elements might change get deprecated over time
+
+### How is Instrument Data Stored?
+
+The data entered by users in an instrument form can be stored in 2 different formats: **SQL table** or **JSON string**. 
+
+JSON data storage makes instrument installation easier by removing the need for table creation in the database and reducing maintenance efforts in the long term. However, SQL storage allows for data amendments to be done directly through SQL commands without the need to parse the JSON data using a tool.
+
+The JSON data string is stored in the `Data` column of the `flag` table. If the SQL mode is enabled, a new table must be created and data is stored into the dedicated instrument table.
+
+>_**Note:** The support for JSON data storage is a new implementation that is meant to deprecate the use of SQL tables in the long term but is currently offered as an alternative only._
+
+## Prerequisites
+
+1. Prepare your **Case Report Forms (CRFs)** including any “lookup” tables used for scoring/normalization and any documentation defining possible relationships between fields for validation.
+2. Determine which **sections of the questionnaire should be included or omitted** from the online version. Typically, unless they pertain to the patients themselves, administrative details included on the original form can be ignored, but instructions that will be helpful for the examiner/patient might be included. 
+3. Decide **how all your fields would map to the form element** types allowed by LORIS
```

Comment:
example template would help here, if this is truly a pre-requisite. Especially for creating LINST forms this is going to be important (given the choices between types of date fields, planning rules, etc. )

Reply:
I will link to existing forms